### PR TITLE
docs: Update tket2-py readme

### PR DESCRIPTION
**Pull request by aborgna-q**
Updates the python README before release, and brings some updates to DEVELOPMENT.md from hugr.

Let's mark the first release version for `release-please` here:

Release-As: 0.1.0a1

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -134,3 +143,50 @@ We accept the following contribution types:
...
+automation to close the release PR and create a new one to avoid conflicts.
+
+### Rust crate release
+
+Rust releases are managed by `release-plz`. This tool will automatically detect
+breaking changes even when they are not marked as such in the commit message,
+and bump the version accordingly.
+
+To modify the version being released, update the `Cargo.toml`,
+CHANGELOG.md, PR name, and PR description in the release PR with the desired version. You may also have to update the dates.
```
Comment (cqc-alec) [inline DEVELOPMENT.md:166]:
```suggestion
`CHANGELOG.md`, PR name, and PR description in the release PR with the desired version. You may also have to update the dates.
```

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -134,3 +143,50 @@ We accept the following contribution types:
...
+### Patch releases
+
+Sometimes we need to release a patch version to fix a critical bug, but we don't want
+to include all the changes that have been merged into the main branch. In this case,
+you can create a new branch from the latest release tag and cherry-pick the commits
+you want to include in the patch release.
+
+You will need to modify the version and changelog manually in this case. Check
+the existing release PRs for examples on how to do this. Once the branch is
+ready, create a [github release](https://github.com/CQCL/hugr/releases/new).
```
Comment (cqc-alec) [inline DEVELOPMENT.md:188]:
```suggestion
ready, create a [github release](https://github.com/CQCL/tket2/releases/new).
```

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -134,3 +143,50 @@ We accept the following contribution types:
...
+The tag should follow the format used in the previous releases, e.g. `hugr-py-v0.1.1`.
```
Comment (cqc-alec) [inline DEVELOPMENT.md:189]:
```suggestion
The tag should follow the format used in the previous releases, e.g. `tket2-v0.1.1`.
```

File: tket2-py/README.md [hunk 4]
```diff
@@ -1,4 +1,31 @@
...
+[![codecov][]](https://codecov.io/gh/CQCL/tket2)
+[![py-version][]](https://pypi.org/project/tket2/)
+
+  [codecov]: https://img.shields.io/codecov/c/gh/CQCL/tket2?logo=codecov
+  [py-version]: https://img.shields.io/pypi/pyversions/tket2
+  [pypi]: https://img.shields.io/pypi/v/tket2
+
+TKET2 is an open source quantum compiler developed by Quantinuum. Central to
+TKET2's design is its hardware agnosticism which allows researchers and
+quantum software developers to take advantage of its state of the art
```
Comment (cqc-alec) [inline tket2-py/README.md:13]: I don't like "state of the art" -- what about "powerful" instead?

File: tket2-py/README.md [hunk 5]
```diff
@@ -1,4 +1,31 @@
...
+compilation for many different quantum architectures.
+
+This is the Python front-end for TKET2, providing a high-level interface for working with quantum circuits.
+See also the Rust library with the same name on [crates.io](https://crates.io/crates/tket2).
+
+
+## Install
+
+Tket2 can be installed via `pip`. Requires Python >= 3.10.
```
Comment (cqc-alec) [inline tket2-py/README.md:22]:
```suggestion
TKET2 can be installed via `pip`. Requires Python >= 3.10.
```

File: tket2-py/README.md (hunk 4 above)
Comment (aborgna-q) [reply tket2-py/README.md:13]: :+1:
I'll change it on the other READMEs too

**Review by cqc-alec — APPROVED** [review]

**Pull request merged**